lib/templates/bootstrap/select.ios.js

```diff
@@ -55,7 +55,7 @@ class CollapsiblePickerIOS extends React.Component {
             }, animationConfig)).start();
             this.setState({isCollapsed: !this.state.isCollapsed});
           }}>
-          <Text style={pickerValue}>
+          <Text numberOfLines={1} style={pickerValue}>
```

Comment:
I would leave the `height` style and make `numberOfLines` configurable, but default to `1`. If the user wants to change one of those parameters, she would have to change both.